### PR TITLE
Update shifts-connector-ukg-known-issues.md

### PR DESCRIPTION
**Pull request by imarquesgil**
Updated on the: A user can’t perform some actions in Shifts in the Teams web app after signing in with a different account section

### CODE REVIEW

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 1]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
...
 
-In this scenario, a caching issue occurs in which the user is signed in to Teams and Shifts on one account and signed in to UKG Dimensions on the other account.
+In this scenario, an issue occurs in which the user B is signed in to Teams and Shifts but user A is still signed in to UKG Dimensions.
 
-To work around this issue, do one of the following actions:
+To mitigate this issue, do one of the following actions:
+
+- Close all the opened browser windows after each user signs out of Teams web app. This needs to happen independently of whether the user has accessed Teams web app through InPrivate window in Microsoft Edge or in Incognito mode in Google Chrome.
+- Clear cookies and site data for the mykronos.com site in the browser after each user signs out of Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md:64]:
```suggestion
- Clear cookies and site data for the mykronos.com site in the browser after each user signs out of the Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 2]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
 
 The correct, expected behavior is a green check mark to indicate that the coworker accepted the request and a red "X" to indicate that the manager denied the request.
 
-## A user can’t perform some actions in Shifts in the Teams web app after signing in with a different account
+## A user B can’t perform some actions in Shifts after another user has signed in to Shifts through the same device
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
## A user can’t perform some actions in Shifts after another user has signed in to Shifts using the same device
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 3]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
...
 
-This issue can occur if a user who has multiple accounts in Teams performs actions in Shifts that require single sign-on (SSO) in UKG Dimensions and that user switches accounts in the Teams web app in the same browser.
+This issue can occur if multiple users, using the same device, try to connect to Teams Shifts app and require to single sign-on (SSO) in UKG Dimensions.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
This issue can occur if multiple users use the same device to connect to the Teams Shifts app and require a single sign-on (SSO) in UKG Dimensions. 
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 4]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
...
 
-## A user can’t perform some actions in Shifts in the Teams web app after signing in with a different account
+## A user B can’t perform some actions in Shifts after another user has signed in to Shifts through the same device
 
-This issue can occur if a user who has multiple accounts in Teams performs actions in Shifts that require single sign-on (SSO) in UKG Dimensions and that user switches accounts in the Teams web app in the same browser.
+This issue can occur if multiple users, using the same device, try to connect to Teams Shifts app and require to single sign-on (SSO) in UKG Dimensions. 
 
-For example, a user signs in to Teams, approves a time off request, and then signs out. The user then signs in to Teams using a different account and tries to perform another action in Shifts that requires SSO in UKG Dimensions.
+For example, a user A signs in to Teams, requests a time off request, and signs out. To perform the request, user A was required to introduce their AAD credentials to verify (SSO). Using the same device, user B signs in to Teams and tries to perform another action in Shifts that requires SSO.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
For example, user A signs in to Teams, requests a time off request, and signs out. To perform the request, user A was required to introduce their Azure AD credentials to verify (SSO). Using the same device, user B signs in to Teams and tries to perform another action in Shifts that requires SSO.
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 5]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
...
 
-In this scenario, a caching issue occurs in which the user is signed in to Teams and Shifts on one account and signed in to UKG Dimensions on the other account.
+In this scenario, an issue occurs in which the user B is signed in to Teams and Shifts but user A is still signed in to UKG Dimensions.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
In this scenario, an issue occurs in which user B is signed in to Teams and Shifts but user A is still signed in to UKG Dimensions.
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md [hunk 6]
```diff
@@ -50,18 +50,19 @@ Currently, if a user creates a swap request with a coworker and the coworker acc
...
 
-To work around this issue, do one of the following actions:
+To mitigate this issue, do one of the following actions:
+
+- Close all the opened browser windows after each user signs out of Teams web app. This needs to happen independently of whether the user has accessed Teams web app through InPrivate window in Microsoft Edge or in Incognito mode in Google Chrome.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
- Close all the opened browser windows after each user signs out of the Teams web app. This needs to happen independently of whether the user has accessed the Teams web app through the InPrivate window in Microsoft Edge or in Incognito mode in Google Chrome.
```

File: microsoft-365/frontline/shifts-connector-ukg-known-issues.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/frontline/shifts-connector-ukg-known-issues.md]:
```suggestion
- Clear cookies and site data for the mykronos.com site in the browser after each user signs out of the Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @imarquesgil, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by daisyfell** [comment]
I'm no longer working on frontline. Assigning to Heidi

**Pull request merged**